elm/Update.elm

```diff
@@ -443,8 +431,42 @@ handleAction action model =
             model ! []
 
 
-changeStage : StageType -> Upd GameModel
-changeStage stagetype model =
+updateAntihunger : Float -> Upd GameModel
+updateAntihunger diff model =
+    -- [note] should this be in sync with UI? Careful with rounding error
+    { model | antihunger = max 0 (model.antihunger + diff) } ! []
+
+
+updateHealth : Float -> GameCtx msg -> Upd GameModel
+updateHealth diff { toGameServer } model =
+    -- [note] should this be in sync with UI? Careful with rounding error
+    let
+        newHealth =
+            max 0 (model.health + diff)
+    in
+    { model | health = newHealth }
+        ! (if newHealth == 0 then
+            [ toGameServer Api.Death ]
+
+           else
+            []
+          )
+
+
+updateHealthWithAntihunger : GameCtx msg -> Upd GameModel
+updateHealthWithAntihunger ctx model =
+    if model.antihunger == 0 then
```

Comment:
I think antihunger shouldn't reset after it hits zero - just stay at zero and keep taking damage every day.